packages/components/src/RichTextEditor/EditableRichTextContent/_docs/EditableRichTextContent.mdx

```diff
@@ -30,13 +30,22 @@ To render the contents of your <LinkTo pageId="components-richtexteditor">RichTe
 The general idea is to use this component in combination with the <LinkTo pageId="components-richtexteditor">RichTextEditor</LinkTo> to toggle between a read only state
 and an editable state.
 
-```jsx
-  const [editMode, setEditMode] = useState(false);
-  const [rteData, setRTEData] = useState();
-
-  if (editMode) return <RichTextEditor defaultValue={rteData} />
+```tsx
+const [editMode, setEditMode] = useState<boolean>(false);
+const [rteData, setRTEData] = useState<EditorContentArray>([]);
 
+if (editMode) {
   return (
-    <EditableRichTextContent content={rteData} />
+    <>
+      <RichTextEditor defaultValue={rteData} />
+      <Button label="Save" primary onClick={handleCancel} />
```

Comment:
Save but cancel? Not sure having the button here serves much to the example though

Reply:
Lol I copied it. Let me take a look and see if I can make it less confusing.

Reply:
I don't think its relevant to the example anyway, safe to remove 😄